docs/userguide/networking/get-started-overlay.md

```diff
@@ -338,6 +339,14 @@ You can try starting a second network on your existing Swarm cluster using Docke
 
 		$ docker-compose --x-networking up -d
 
+7. Obtain the IP address of the VM for the Swarm master.
+
+		$ docker-machine ip mhs-demo0
+
+8. Open a web browser on this IP address and you should see that the web app has
```

Comment:
I think 7. and 8. could be combined, using (e.g.) "Verify your application is working" as title. Then explain the steps to obtain the IP address, and to open the address in a webbrowser